README.md

```diff
@@ -71,10 +75,10 @@ It's possible to specify configuration file as a flag, default is `gorss.conf`.
     "OPMLFile": "../example_ompl.xml",
     "feeds": [
         "https://news.ycombinator.com/rss",
-        "https://www.sweclockers.com/feeds/nyheter",
-        "https://www.reddit.com/r/homeassistant/.rss",
-        "https://www.reddit.com/r/golang/.rss",
-        "https://www.reddit.com/r/programming/.rss"
+        ("url": "https://www.sweclockers.com/feeds/nyheter", "name": "Swedish Overclocking"},
+        ("url": "https://www.reddit.com/r/homeassistant/.rss", "name": "Home Assistant"},
+        ("url": "https://www.reddit.com/r/golang/.rss"},
```

Comment:
I think there's a typo here on these 3 lines with `(` instead of `{`.

Reply:
Good catch. I fixed that directly in master branch.